sw/device/tests/BUILD

```diff
@@ -2274,6 +2274,39 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "spi_device_smoketest",
+    srcs = ["spi_device_smoketest.c"],
+    cw310 = new_cw310_params(
+        bitstream = "//hw/bitstream/hyperdebug:test_rom",
```

Comment:
Reminder that we need to rebase this PR and remove this line now that the sival exec_env uses hyperdebug: #20557

Reply:
Fixed.